java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/SinkStreamObserver.java

```diff
@@ -33,6 +33,10 @@
 public class SinkStreamObserver implements StreamObserver<ConnectorServiceProto.SinkStreamRequest> {
     private SinkBase sink;
 
+    private String sinkType;
```

Comment:
Maybe we should use `connectorType` here to avoid ambiguity. `sinkType` stands for a sink's append-only attribute in our code, while `connectorType` refers to what the downstream connector is (e.g. JDBC, MySQL).